src/i_flmusic.c

```diff
@@ -275,12 +278,31 @@ static void I_FL_ShutdownMusic(void)
 
 int I_FL_DeviceList(const char* devices[], int size)
 {
-    if (size > 0)
+    int i;
+    glob_t *glob;
+
+    sf_paths = malloc(size * sizeof(*sf_paths));
+
+    glob = I_StartMultiGlob("soundfonts", GLOB_FLAG_NOCASE|GLOB_FLAG_SORTED,
```

Comment:
How about `M_DirName(soundfont_path)` here?

Reply:
Not sure about it. Maybe we should introduce new "soundfonts_dir" config variable?

Reply:
Yes, okay, to separate it from the "soundfonts_path" variable. However, if we go that way you outlined here, we should just drop the "soundfonts_path" variable altogether.

Reply:
Actually, we will need to support a list of directories. In Debian, soundfonts are installed into `/usr/share/sounds/sf2` and `.../sf3`, respectively:

https://packages.debian.org/bullseye/all/timgm6mb-soundfont/filelist
https://packages.debian.org/bullseye/all/musescore-general-soundfont-small/filelist

Reply:
`.sf3` uses lossy compression and loads slowly, so we can ignore `/usr/share/sounds/sf3` 😉
How will we do it? Multiple configuration variables, or use some kind of separator `"path1;path2"`?

Reply:
We define a `PATH_SEPARATOR` in `src/doomtype.h` for this case.

Reply:
We actually use it in `d_iwad.c:ADddIWADPath()`.